packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -279,7 +279,14 @@ export const createWebpackUtils = (
       return {
         loader: require.resolve(`css-loader`),
         options: {
-          url: false,
+          url: function (url: string): boolean {
+            // When an url starts with /
+            if (url.startsWith(`/`)) {
```

Comment:
What happens when someone uses fully-qualified URLs? I.e. `url("https://example.com/some-image.jpg")`

Reply:
I did not test :) good one!

Reply:
And similar case (just without protocol): `url("//example.com/some-image.jpg")`

Reply:
This one starts with `/` 😄

Reply:
🤦